- added global delay and polarisation leakage plots to crosscal

adebahr opened the pull request:
- will include them in report after this and the selfcal qa has been merged

Commit:
- added global delay and polarisation leakage plots to crosscal
- will include them in report after this and the selfcal qa has been merged

eakadams commented:
The generation of all plots except polarization leakage are commented out. This needs to be fixed before merging.

Commit:
- added creation of plots for crosscal polarisation
- still need to add them to the report
- commented old plots back in
- needs a polcal specified now
